tools/targets/STM32_gen_PeripheralPins.py

```diff
@@ -1329,10 +1460,13 @@ def parse_BoardFile(fileName):
 
     # Add some hardcoded check
     if "J01_" in board_file_name:
-        print ("J01_Discovery_STM32F4-DISCO-AudioPack_STM32F407V_Board not parsed")
+        print("J01_Discovery_STM32F4-DISCO-AudioPack_STM32F407V_Board not parsed")
+        quit()
```

Comment:
We should use `sys.exit` rather than `quit`. `quit` is meant for terminating an interactive session and relies on the `site` module being loaded, which (in CPython) is when the interpreter is started. It might not work as expected in all cases in a script.